.github/workflows/security.yml

```
@@ -0,0 +1,19 @@
+name: Perform Weekly Security Audit with ZAProxy
+# Use ZAP Proxy to perform a full scan of the production site.
+# Scan automatically opens an issue after completion
+# with results of the audit.
+
+on:
+  schedule:
+    # 17:30 UTC (5:30 pm) on Fridays
+    - cron: '30 17 * * 5'
```

I'd prefer Monday at Midnight so that I can see any new alerts first thing in the week.

```suggestion
    - cron: '0 0 * * 1'
```